datastore/store.go

```diff
@@ -200,6 +200,11 @@ func (db *CachedDB) GetAtx(id types.ATXID) (*types.ActivationTx, error) {
 	return atx, nil
 }
 
+// Previous retrieves the list of previous ATXs for the given ATX ID.
+func (db *CachedDB) Previous(id types.ATXID) ([]types.ATXID, error) {
+	return atxs.Previous(db, id)
+}
```

Comment:
It can't be inlined, because it's used in the V1 activation GRPC service, abstracted by the `atxProvider` interface. But we will retire V1 soon anyway.